archai/nlp/train.py

```diff
@@ -954,7 +954,7 @@ def distributed_model(args, model, device):
                                              device_ids=[args.local_rank],
                                              output_device=args.local_rank,
                                              broadcast_buffers=False,
-                                             find_unused_parameters=utils.is_debugging(),
+                                             find_unused_parameters=True,

```

Comment:
For some reason, as utils.is_debugging() defaults to `False`, it is breaking the distributed training for architectures that use the `ProjectedAdaptiveLogSoftmax`.

I have traced back the problem and it dates back to this commit: https://github.com/microsoft/archai/commit/05cfe0f070b4ea86ef918bc3771c65d6f3b4a42d.

I think that we have not trained a full distributed model with `ProjectedAdaptiveLogSoftmax` since that date and thus we haven't seen the error.